lib/travis/build/script/elm.rb

```diff
@@ -0,0 +1,183 @@
+module Travis
+  module Build
+    class Script
+      class Elm < NodeJs
+        # Default NodeJS version to install
+        DEFAULT_NODE_VERSION = '10.13.0'
+
+        DEFAULTS = {
+          elm: '0.19.0',
+          elm_test: '0.19.0',
+          elm_format: '0.19.0'
+        }
+
+        ELM_TEST_REQUIRED_NODE_VERSION = '6.0.0'
+
+        def export
+          super
+          sh.export 'TRAVIS_ELM_VERSION', elm_version, echo: false
+          sh.export 'TRAVIS_ELM_TEST_VERSION', elm_test_version, echo: false
+          sh.export 'TRAVIS_ELM_FORMAT_VERSION', elm_format_version, echo: false
+        end
+
+        def configure
+          super
+
+          config[:node_js] ||= DEFAULT_NODE_VERSION
+        end
+
+        def announce
+          super
+          sh.cmd 'elm --version', echo: true
+          sh.cmd 'elm-test --version', echo: true
+
+          # elm-format doesn't have --version,
+          # but the first line of `elm-format --help` prints the version
+          sh.cmd 'elm-format --help | head -n 1', echo: true
+        end
+
+        def setup
+          super
+
+          sh.echo 'Elm for Travis-CI is not officially supported, ' \
+          'but is community maintained.', ansi: :green
+          sh.echo 'Please file any issues using the following link',
+            ansi: :green
+          sh.echo '  https://github.com/travis-ci/travis-ci/issues' \
+            '/new?labels=community:elm', ansi: :green
+          sh.echo 'and mention \`@avh4\`, \`@lukewestby\`, \`@stoeffel\` and \`@rtfeldman\`' \
+            ' in the issue', ansi: :green
+
+
+          sh.if '! -d sysconfcpus/bin' do
+            install_sysconfcpus
+          end
+
+          sh.fold 'install.elm' do
+            install_elm
```

Comment:
I suggest adding a colored `echo` command above this, which acts as a label for the `install.elm` fold. (Currently, the fold looks indistinguishable from other lines.)

Reply:
Added in e386ff219defbe58bbe931cc0b66b57e866d818c